### DISCUSSION TITLE
Add buttons for scroll lock, fit screen, zoom+/- on main screen

### DISCUSSION THREAD

**Discussion by Zamrod** [encounterplus/encounterplus#493] [Ideas]
Add a button somewhere on the main interface for scroll lock so that you can easily toggle it on and off without going deep into a menu.

**Comment by Splinter714** [comment]
Consolidating similar requests here

> @Duseylicious #214
> i use the "fit screen" option often. It'd be nice if there was a "fit screen" button right next to zoom buttons on the battle map, to be able to quickly fit a newly loaded map. Bonus for adding a keyboard shortcut to do the same. (command-0 is a common "fit screen" shortcut, with command + being zoom in and command - being zoom out.)

**Comment by jurex** [comment]
scroll lock added
